src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(calculateStatistics(fromFileName), toFileName);
+    }
+
+    private String calculateStatistics(String fromFileName) {
+        final int AcT = 0;
+        final int DigiT = 1;
```

Comment:
The constant names `AcT` and `DigiT` do not follow Java naming conventions. Constants should be in uppercase letters with words separated by underscores. For example: `ACTION_INDEX` and `DIGIT_INDEX`.